modules/ocf_irc/files/anope.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=A set of IRC Services
+After=network-online.target inspircd.service
+
+[Service]
+User=irc
+Group=irc
+Restart=always
+# Run ExecStartPre with root permissions to be able to create pid directory
+PermissionsStartOnly=true
+ExecStartPre=/bin/mkdir -p /var/run/anope/
```

Comment:
I mean that works too, it seems like `/var/run/anope/anope.pid` is where it tries to make the PID (with no option I could find to change the PID directory??), so that's why I chose this. If it's a symlink then that's fine too, although I'd prefer to keep it pointing to the actual path that anope will use for creating the PID file.